src/Microsoft.TemplateEngine.Abstractions/IPostAction.cs

```diff
@@ -1,23 +1,44 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
+
 using System;
 using System.Collections.Generic;
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
     public interface IPostAction
     {
-        string Description { get; }
+        /// <summary>
+        /// Gets the description of the post action.
+        /// </summary>
+        string? Description { get; }
 
+        /// <summary>
+        /// Gets the identifier of the action that will be performed.
+        /// Note that this is not an identifier for the post action itself.
+        /// </summary>
         Guid ActionId { get; }
 
+        /// <summary>
+        /// Gets a value indicating wheather the template instantiation should continue
+        /// in case of an error with this post action.
+        /// </summary>
         bool ContinueOnError { get; }
 
+        /// <summary>
+        /// Gets the arguments for this post action.
+        /// </summary>
         IReadOnlyDictionary<string, string> Args { get; }
 
-        string ManualInstructions { get; }
+        /// <summary>
+        /// Gets the instructions that should be manually performed by the user
+        /// as part of this post action.
+        /// Manual instructions are used when the host does not have the associated post action processor implemented.
+        /// </summary>
+        string? ManualInstructions { get; }
 
-        string ConfigFile { get; }
+        string? ConfigFile { get; }
```

Comment:
This doesn't appear to be used, so I removed it all together (internal team discussion also didn't yield concrete results on what this was for). You did ask me if this could be removed, right? For some reason, I had found some ties that prevented me from deleting it. Anyway, its gone now.